Langs/040A.lang

```diff
@@ -1,7 +1,7 @@
 [MetaData]
 Language=Spanish - Spain
 Translator=Konata400, JeffersonRussell
```

Comment:
@Konata400 Do you approve these changes?

Reply:
Yes